src/turbopuffer.ts

```diff
@@ -411,6 +429,35 @@ export class Namespace {
       })
     ).body!;
   }
+
+  /**
+   * Returns the current schema for the namespace.
+   * See: https://turbopuffer.com/docs/schema
+   */
+  async schema(): Promise<Record<string, Schema>> {
+    const res = (await this.client.http.doRequest({
+      method: "GET",
+      path: `/v1/namespaces/${this.id}/schema`,
+      retryable: true,
+    })).body!;
+
+    for (const value of Object.values(res)) {
+      const val = value as Schema[string];
+      if (
+        typeof val.full_text_search === 'object' &&
+        val.full_text_search !== null
+      ) {
+        val.full_text_search = pick(val.full_text_search, [
+          'language',
+          'stemming',
+          'remove_stopwords',
+          'case_sensitive'
+        ]);
```

Comment:
as discussed in person we should just return everything, the python client is not intentionally picking specific fields